agent/heuristics/heuristic_type.py

```diff
@@ -0,0 +1,19 @@
+from __future__ import annotations
+
+from enum import Enum
+from numbers import Number
+
+from agent.heuristics.heuristic import Heuristic
+from core.board import Board
+from core.color import Color
+
+
+class HeuristicType(Enum):
+    WEIGHTED_NORMALIZED = "Weighted Normalized"
+    WEIGHTED = "Weighted"
+
+    def call(self, board: Board, player: Color) -> Number:
+        return {
+            HeuristicType.WEIGHTED_NORMALIZED: lambda: Heuristic.weighted_normalized(board, player),
+            HeuristicType.WEIGHTED: lambda: Heuristic.weighted(board, player),
+        }[self]()
```

Comment:
New heuristics for testing can be added here easily. Add a member to the Enum, set its value as the display name and then add it to the call function in the same format as above. This will allow us to easily test new heuristics / weightings vs each other.